libraries/linters/snippet_comparison.py

```diff
@@ -0,0 +1,264 @@
+'''
```

Comment:
Should this file be here?  It looks almost the same as the tn_linter below.

Reply:
This is the actual snippet comparison. Both the tn and ulb linters will call it, once they have each determined the actual book/chapter/verses that needs to be compared. There is some commonality between this snippet_comparison and the linters, but each has a slightly different task to perform, and I wanted the actual snippet compare in 1 place only, since it is called by both the other linters.